### PR TITLE
refactor(explorer): remove unused code

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Removed dead code in Explorer

  # To Test

Explorer must work as before

### CODE REVIEW

File: apps/explorer/src/hooks/useSolversInfo.ts
Comment (alfetopito) [inline apps/explorer/src/hooks/useSolversInfo.ts:1]: Let's keep this, it's part of the solvers page that we'll have one day 🤞

**Review by alfetopito — APPROVED** [review]

File: apps/explorer/src/test/utils/toCsv.spec.ts
Comment (anxolin) [inline apps/explorer/src/test/utils/toCsv.spec.ts:1]: Are we sure we want to remove the CSV functionality. Could it come handy if we want to implement in the explorer some CSV export?

**Review by anxolin — APPROVED** [review]
Cool! Did you use a tool to find unused code, or you just took the scissors and deleted it?

**Comment by shoom3301** [comment]
>Cool! Did you use a tool to find unused code, or you just took the scissors and deleted it?

Yes, I mostly used IDEA to find a file/function usage

image

File: apps/explorer/src/hooks/useSolversInfo.ts
Comment (shoom3301) [reply apps/explorer/src/hooks/useSolversInfo.ts:1]: Sure! ADded it back with a comment to not delete it

File: apps/explorer/src/test/utils/toCsv.spec.ts
Comment (shoom3301) [reply apps/explorer/src/test/utils/toCsv.spec.ts:1]: YAGNI :) 
I will find the function once we need it again

Comment (alfetopito) [reply apps/explorer/src/test/utils/toCsv.spec.ts:1]: Thought the same because I'm proud of my little csv baby, but I agree. Let's kill it.

**Comment by elena-zh** [comment]
LGTM!

File: apps/explorer/src/test/utils/toCsv.spec.ts
Comment (anxolin) [reply apps/explorer/src/test/utils/toCsv.spec.ts:1]: ahaha, it was working great I remember. YAGNI is a great principle though :)

**Pull request merged**